backend/dataall/modules/metadata_forms/services/metadata_form_service.py

```diff
@@ -69,3 +80,51 @@ def get_home_entity_name(metadata_form):
                 return EnvironmentRepository.get_environment_by_uri(session, metadata_form.homeEntity).name
         else:
             return ''
+
+    @staticmethod
+    def get_metadata_form_fields(uri):
+        with get_context().db_engine.scoped_session() as session:
+            return MetadataFormRepository.get_metadata_form_fields(session, uri)
+
+    @staticmethod
+    def get_metadata_form_field_by_uri(uri):
+        with get_context().db_engine.scoped_session() as session:
+            return MetadataFormRepository.get_metadata_form_field_by_uri(session, uri)
```

Comment:
Sorry, but no)))
In this case we would have to either create a session in resolver, which we tried to eliminate by lots of refactoring, or create session in Repository, which is also no good.
Also, in next PRs this method is the place, where the access right will be checked.
So we really do need this method.

Reply:
The suggestion is not to create session in resolver or repository - I was suggesting we delete the `def get_metadata_form_field_by_uri` from `MetadataFormService` since it is not being used anywhere other than as part of `MetadataFormService.delete_metadata_form_field` and feel unnecessary

Instead I was thinking `delete_metadata_form_field` would become:

    @staticmethod
    def delete_metadata_form_field(uri, fieldUri):
        with get_context().db_engine.scoped_session() as session:
            mf = MetadataFormRepository.get_metadata_form_field_by_uri(fieldUri)
            return session.delete(mf)
```

still creating session in service and passing to use in repository as we do it elsewhere